tests/test-war-integration/pom.xml

```diff
@@ -45,8 +45,51 @@
       <artifactId>jackson-databind</artifactId>
       <version>2.8.7</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-logging</artifactId>
+      <version>1.2.1</version>
+    </dependency>
   </dependencies>
 
+  <profiles>
+    <profile>
+      <id>local-docker-build</id>
```

Comment:
The profile is activated by default, it is disabled during the `cloudbuild` step (to avoid creating a Docker image locally). This profile is already present in the other test applications.